src/Components/ProductCard/index.js

```diff
@@ -1,20 +1,71 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
+import { useState, useEffect } from 'react';
 import classnames from 'classnames';
 
 import moneyFormatter from '../../utilities/moneyFormatter';
 import './style.scss';
 
 const ProductCard = ({ product, className }) => {
+  const [selectedVariant, setSelectedVariant] = useState(false);
+  const [selectedColor, setSelectedColor] = useState(false)
+
   const onClick = () => {
     sessionStorage.productDetails = JSON.stringify(product);
     window.location = `/product-details?product=${product.productID}`;
   }
+
+  const colorMetaName = 'Color';
```

Comment:
@hemant-cbotics if we are only to consider english, we can for the following keywords:
- Color
- color
- colour
- Colour
Simply makes sure it will work for both `color` and `colour` and should be case insensitive

Reply:
@Nziranziza - totally agree with you on that one! I'll update accordingly.